.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Should VSCode settings be pushed to git?

Reply:
No, they will have to add it to gitIgnore, I will tell them now

Reply:
Promise try and fix conflicts on tracking.html, create a branch for it and commit it back to develop branch